shell/plugins/steve/steve-pagination-utils.ts

```diff
@@ -499,6 +505,10 @@ class StevePaginationUtils extends NamespaceProjectFilters {
               // Check if the API supports filtering by this field
               this.validateField(validateFields, schema, field.field);
 
+              // we're just checking that the field exists, so there's no value
+              if (field.exists) {
+                return field.field;
+              }
               const encodedValue = encodeURIComponent(field.value);
```

Comment:
```suggestion
              const encodedValue = encodeURIComponent(field?.value || '');
```

to fix the TS complain